tests/EdFi.Admin.DataAccess.IntegrationTests/Models/UserContextTestBase.cs

```diff
@@ -17,14 +20,22 @@ public abstract class UserContextTestBase : TestFixtureBase
     {
         private TransactionScope _transaction;
 
-        protected string ConnectionString { get; private set; }
+       protected string ConnectionString { get; private set; }
 
         [OneTimeSetUp]
         public void OneTimeSetUp()
         {
-            using (var usersContext = new SqlServerUsersContext())
+            var config = new ConfigurationBuilder()
+                  .SetBasePath(Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, @"..\..\..")) + "\\")
```

Comment:
modify the project file and have the appsettings.json file copied to the the output path, then you can remove these relative paths

Reply:
We should be using TestContext.CurrentContext.TestDirectory for the base path